### PR TITLE
perf(aad-manifest): add deploy aad manifest command in CLI

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: packages/fx-core/src/plugins/resource/aad/index.ts [hunk 1]
```diff
@@ -149,6 +152,31 @@ export class AadAppForTeamsPlugin implements Plugin {
...
+
+    if (
+      stage === Stage.deploy &&
+      (ctx.answers?.platform === Platform.CLI_HELP || ctx.answers?.platform === Platform.CLI)
+    ) {
+      const node = new QTreeNode({
+        name: Constants.INCLUDE_AAD_MANIFEST,
+        type: "singleSelect",
+        staticOptions: ["yes", "no"],
+        title: "Whether to deploy aad manifest",
```
Comment (KennethBWSong) [inline packages/fx-core/src/plugins/resource/aad/index.ts:171]: Need to localize?

File: packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts [hunk 2]
```diff
@@ -289,13 +292,27 @@ export async function getQuestions(
     } else {
       plugins = getAllV2ResourcePlugins();
     }
-    plugins = plugins.filter((plugin) => !!plugin.deploy && plugin.displayName !== "AAD");
-    if (plugins.length === 0 && inputs.skipAadDeploy !== "no") {
+
+    let filterAadPlugin = true;
+    if (
+      isAadManifestEnabled() &&
+      (inputs.platform === Platform.CLI_HELP || inputs.platform === Platform.CLI)
```
Comment (KennethBWSong) [inline packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts:299]: What is this line for?

File: packages/fx-core/src/plugins/resource/aad/index.ts (hunk 1 above)
Comment (SLdragon) [reply packages/fx-core/src/plugins/resource/aad/index.ts:171]: Sure, thanks!

File: packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts (hunk 2 above)
Comment (SLdragon) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts:299]: We do not filter questions of AAD when cli or cli_help

File: packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts [hunk 3]
```diff
@@ -289,13 +292,27 @@ export async function getQuestions(
...
+    ) {
+      filterAadPlugin = false;
+    }
+
+    if (filterAadPlugin) {
```
Comment (1yefuwang1) [inline packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts:304]: Perhaps it's better to merge this if block with the if block on Line 297

**Review by Alive-Fish — APPROVED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts (hunk 3 above)
Comment (SLdragon) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts:304]: OK, thanks

**Review by KennethBWSong — APPROVED** [review]

**Review by 1yefuwang1 — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by nliu-ms — APPROVED** [review]

**Pull request merged**